scripts/src/image-viewer.js

```diff
@@ -1,3 +1,5 @@
+/* global image_source */
```

Comment:
where variables are being declared in html files I have declared them as globals. Ideally they'd be passed as data attributes rather than being declared in separate files.